comments/serializers.py

```diff
@@ -1,12 +1,19 @@
+from typing import Dict
+
 from rest_framework import serializers
 
 from comments.models import Comment
-from users.serializers import UserPublicSerializer
 
 
 class CommentSerializer(serializers.ModelSerializer):
-    users = UserPublicSerializer(many=True, read_only=True)
-
     class Meta:
         model = Comment
-        fields = ["text", "post_id", "users", "timestamp"]
+        fields = ["text", "post", "timestamp"]
+
+    def create(self, data: Dict) -> Comment:
+        request = self.context.get("request")
+        comment = Comment.objects.create(
```

Comment:
`.objects.create` calls `.save()`. You don't have to call it explicitly

Reply:
Removed